README.md

```diff
@@ -46,10 +46,10 @@ There are multiple ways to get involved and learn more about our work. That's ho
 - **Specific code meeting (Open)**
   - meeting hosted on Discord
   - join updates, demos, Q&A's, discussions and the coordination of each work package
-    1. Demand creation and prediction meeting, every Wednesday 21:00 UTC+0, <a href="https://drive.google.com/file/d/1OBGBwSWeuYzhssb6tDxyAPIzz3uUhsnv/view?usp=sharing" >download .ics</a>
```

Comment:
I just noticed that there are some refuses when resolving the conflicts.
Please restore the previous version

Reply:
Sorry,  it seems I had to do double check when merging! Should be fixed now.

Reply:
no problem, it happens :)